app/tests/spec/models/account.js

```diff
@@ -684,5 +684,38 @@ function (chai, sinon, p, Constants, Assertion, ProfileClient,
           });
       });
     });
+
+    describe('completePasswordReset', function () {
+      it('completes the password reset, signs the user in', function () {
+        account.set('email', EMAIL);
+        account.set('password', PASSWORD);
+        var token = 'token';
+        var code = 'code';
+
+        sinon.stub(fxaClient, 'signIn', function () {
+          return p({});
+        });
+        sinon.stub(fxaClient, 'completePasswordReset', function () {
+          return p(true);
+        });
+        sinon.stub(relier, 'isDirectAccess', function () {
+          return false;
+        });
+
+        return account.completePasswordReset(token, code, relier)
+          .then(function () {
+            assert.isTrue(fxaClient.completePasswordReset.calledWith(
+                EMAIL, PASSWORD, token, code));
```

Comment:
We seem to indent 2 spaces rather than 4 for long argument lists in other places.